reference/forms/types/entity.rst

```diff
@@ -103,6 +103,29 @@ then you can supply the ``choices`` option directly::
 Field Options
 -------------
 
+choice_label
+~~~~~~~~
```

Comment:
The underline must match the length of the headline (so you need to add four `~` characters).